.all-contributorsrc

```diff
@@ -43,6 +43,17 @@
         "test"
       ]
     },
+    {
+      "login": "xavidop",
+      "name": "Xavier Portilla Edo",
+      "avatar_url": "https://avatars.githubusercontent.com/u/4416096?v=4",
+      "profile": "https://xavidop.me",
+      "contributions": [
+        "code",
+        "doc",
+        "test"
+      ]
+    },
```

Comment:
this will be added in a separate PR https://github.com/TheFireCo/genkit-plugins/pull/172
```suggestion
```

Reply:
@davidoort this is fixed!